draft1.md

```diff
@@ -47,15 +52,18 @@ For each dimension `i` in the embedded space:
   Let V2_i be W_i projected in embedded space S_B
   Let V2_i be the i-th dimension of the transformation matrix T
 ```
-** *NOTE:** to convert $V1_i$ back into vocabulary space, we take the dot product of the embedded vector and the inverse of embedding matrix.
+** *NOTE:** to convert $V1_i$ back into vocabulary space, we take the product of the embedded vector and the inverse of the embedding matrix.
+(**EGP: Note note: dot product of a vector and a matrix does not exist, so I removed "dot"; or did you mean something special?**)
```

Comment:
In python, I meant we do: `Minv.dot(v)`, where `Minv` is the inverse embedding matrix, and `v` is the embedded vector. (I've omitted a transpose, because we only transpose to make the dimensions match. I can just write "product" and leave the detail of the dot product buried in the code.